recipe/meta.yaml

```diff
@@ -37,7 +37,13 @@ requirements:
     - rasterio
     - singledispatch
     - sqlalchemy
-    - xarray
+    - xarray >=0.9
+    - celery >=4
+    - paramiko
+    - redis  # [not win]
```

Comment:
I think we can change this to `redis-py` and remove the `# [not win]` annotation?